### ISSUE TITLE
Is it possible to add REST request headers in serenity log?

### ISSUE THREAD

**Issue by dsmt** [serenity-bdd/serenity-core#198]
Can you add request headers in Serenity logs?

**Comment by YamStranger** [comment]
Hello @dsmt. Thanks for this issue!

In serenity we use internally [rest-assured](https://github.com/jayway/rest-assured). We add request/response information only in aggregation reports, logging of body, headers, cookies, and so on already implemented in it [rest-assured](https://github.com/jayway/rest-assured) and can be enabled with some configuration according to their [manual](https://github.com/jayway/rest-assured/wiki/Usage).  

Here you can find all necessary information about [rest-assured Logging](https://github.com/jayway/rest-assured/wiki/Usage#logging) 

P.S. By default .log().... prints to stdout, so if you run your build (tets) with --info you find all necessary info.

**Comment by YamStranger** [comment]
Hello @dsmt!

Lets close this issue, because solution was find.
For logging you can use .log(). 

Yes it produces some issues for now and we have from you issue: 
https://github.com/serenity-bdd/serenity-core/issues/185, but those approach can be used according to my comment 

> workaround:

        RequestSpecification spec = rest().given().contentType("application/json")
                .content(jsonPet);
        spec.log().all();
        spec.post("http://petstore.swagger.io/v2/pet"); 

What you think about it?

**Comment by dsmt** [comment]
@YamStranger, it is another issue with the mistake in the description. I meant if it is possible to place headers in Serenity reports?

**Comment by YamStranger** [comment]
@dsmt thanks for clarification!

**Comment by YamStranger** [comment]
Hi @dsmt! 

Now it is not possible, but may be it will be implemented in future.

**Comment by YamStranger** [comment]
depends on #355

**Comment by YamStranger** [comment]
Hello @dsmt 
You request is implemented and be checked in serenity-core 1.1.31. 

Now rest reports include headers and cookies (if they are not empty) of request and response:

![after1](https://cloud.githubusercontent.com/assets/1667699/14665160/7f728d04-06d9-11e6-9dac-38f52076d990.png)
![after2](https://cloud.githubusercontent.com/assets/1667699/14665161/7f95af5a-06d9-11e6-9025-44a7e298d87c.png)


Hope, now you will much more happy with serenity rest ) 

please close this ticket

Thanks a lot!

**Comment by dsmt** [comment]
Thanks a lot! It was long-expected feature!
